tc/sgx/trusted_worker_manager/enclave/work_order_processor.h

```diff
@@ -27,6 +27,11 @@ namespace tcf {
         class WorkOrderProcessor {
         public:
                 WorkOrderProcessor() {}
+                ~WorkOrderProcessor() {
+                    // Sanitize class members storing secrets
+                    worker_encryption_key.clear();
```

Comment:
Are these class instance variables?

Reply:
that's correct. i am clearing the variables whose values are sensitive and need to be confidential.